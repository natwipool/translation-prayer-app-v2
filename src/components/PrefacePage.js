import React from 'react';

const PrefacePage = () => (
  <div>
    <div className="page-header">
      <div className="content-container-body">
        <h2 className="page-header__title">คำนำ</h2>
      </div>
    </div>
    <div className="content-container-body">
      <img className="cover-image" src="/images/preface.jpg" />
      <div className="text-container">
        <p>
          การสวดมนต์ เป็นกิจวัตรที่สำคัญอย่างหนึ่งของพุทธศาสนิกชน
          ซึ่งถือปฏิบัติติดต่อกันมาเป็นเวลายาวนาน โดยเฉพาะ พระภิกษุ สามเณร
          จะต้องปฏิบัติกันเป็นประจำทั้งเวลาเช้าและเวลาเย็น โดยมิขาด เมื่อถึงเวลา
          วัดต่างๆ จะตีสัญญาณระฆังทั้งเช้าและเย็นจนเป็นที่รู้และคุ้นเคยกันทั้ง
          พระภิกษุ สามเณร และชาวบ้าน
        </p>
        <p>
          การสวดมนต์ไหว้พระ เป็นการสืบต่ออายุพระศาสนา
          และเป็นการอบรมจิตใจของผู้สวดให้มีสมาธิ ไม่ฟุ้งซ่าน
          และเป็นการกล่อมเกลาอัธยาศัยให้ปราณีต เป็นการพัฒนาจิตใจให้มีคุณธรรม
          ยิ่งๆ ขึ้นไป อย่างไรก็ตาม การไหว้พระสวดมนต์ โดยทั่วไปนั้น
          ไม่เฉพาะแต่พระสงฆ์และสามเณรเท่านั้น แม้แต่คฤหัสถ์ทั่วไปก็นิยมสวดมนต์
          ไหว้พระก่อนนอนหรือทำวัตรเช้า-เย็น กันอย่างกว้างขวาง เพราะถือเป็นบุญ
          เป็นกุศล เป็นการปฏิบัติธรรมอย่างหนึ่ง
        </p>
        <p>
          เว็บสวดมนต์แปลนี้ ได้รวบรวมบทสวดมนต์แปล พร้อมทั้งเสียงสวดมนต์
          ฉบับวัดหนองป่าพง เพื่อเป็นเครื่องมืออำนวยความสะดวก
          เพื่อจะได้เกิดประโยชน์แก่ผู้สนใจ และผู้ปฏิบัติธรรมทุกๆท่าน สุดท้ายนี้ก็ขอ อนุโมทนาบุญกับทุกท่านด้วยครับ
        </p>
      </div>
    </div>
  </div>
);

export default PrefacePage;
