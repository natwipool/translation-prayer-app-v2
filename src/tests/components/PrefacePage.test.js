import React from 'react';
import { shallow } from 'enzyme';
import PrefacePage from '../../components/PrefacePage';

test('should render PrefacePage correctly', () => {
  const wrapper = shallow(<PrefacePage />)
  expect(wrapper).toMatchSnapshot();
});