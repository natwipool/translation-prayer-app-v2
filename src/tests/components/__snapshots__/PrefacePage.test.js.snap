// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PrefacePage correctly 1`] = `
<div>
  <h1>
    คำนำ
  </h1>
</div>
`;
